misc/userscripts/qute-pass

```diff
@@ -140,9 +159,13 @@ def find_pass_candidates(domain, unfiltered=False):
             split_path = pass_path.split(os.path.sep)
             for secret in secrets:
                 secret_base = os.path.splitext(secret)[0]
-                if not unfiltered and domain not in (split_path + [secret_base]):
+                idna_domain = idna_encode(domain)
+                idna_split_path = [idna_encode(part) for part in split_path]
```

Comment:
And this one should go outside the `for secret in secrets` loop as the split_path doesn't change between different secrets under the same folder.